### PR TITLE
Ticket 6

### PR DESCRIPTION
**Pull request by Cartesianist**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -63,6 +63,89 @@ describe('/api/articles/:article_id', () => {
...
+                });
+        });
+        test('400: Invalid input', () => {
+            return request(app)
+                .patch('/api/articles/1')
+                .send({})
+                .expect(400)
+                .then(({ body }) => {
+                    expect(body).toEqual({ msg: "Invalid input" });
+                    return db.query("SELECT * FROM articles WHERE article_id=1");
```
Comment (HMelesi) [inline __tests__/app.test.js:106]: I think either use the article_id variable or hardcode in 1 in these database checks in your test (you've done it differently on line 86). I would argue just to hardcode it like you've done here.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -63,6 +63,89 @@ describe('/api/articles/:article_id', () => {
...
+                        article_id: 1,
+                        title: "Living in the shadow of a great man",
+                        topic: "mitch",
+                        author: "butter_bridge",
+                        body: "I find this existence challenging",
+                        created_at: expect.anything(),
+                        votes: 100
+                    });
+                });
+        });
```
Comment (HMelesi) [inline __tests__/app.test.js:144]: You need to add a test for if you try to update an article with:

- invalid id - e.g. '/api/articles/invalid'
- not found id - e.g. '/api/articles/9999999999'

File: controllers/articles.controllers.js [hunk 3]
```diff
@@ -8,3 +8,11 @@ exports.getArticle = (req, res, next) => {
         })
         .catch(next);
 };;
+
+exports.patchArticle = (req, res, next) => {
+    const votes = req.body.inc_votes;
```
Comment (HMelesi) [inline controllers/articles.controllers.js]: You could destructure {inc_votes} here like you have done for article_id too

**Review by HMelesi — CHANGES_REQUESTED** [review]
A couple of tests to add and comments but looking good! Submit a new nchelp pr when they're done :)

**Review by xViolaine — APPROVED** [review]

**Pull request merged**